src/mysqloperator.tex

```diff
@@ -69,14 +75,16 @@ \subsection{Design Patterns}
 \texttt{kubectl patch} command. From the operator’s point of view, an \textbf{Update} event is
 received and the corresponding operator handler is called by the controller.
 
-We use the controller and operator patterns to manage resources related to both 
+We use the controller and operator patterns to manage resources related to both
 MySQL servers and backup jobs. A high level overview of the project design is
 illustrated in Figure \ref{fig:design}. During operator startup, two
 CustomResourceDefinitions (Figure \ref{fig:design} A) are registered into the Kubernetes API.
 From this point on, users can define their own custom resource instances (Figure \ref{fig:design} B) ---
-MySQLClusters (ex. C1, C2) and MySQLBackupSchedules (ex. B1, B2). Each kind of
+MySQLClusters (C\#1, C\#2) and MySQLBackupSchedules (B\#1, B\#2). Each kind of
 custom resource has its corresponding controller listening and reacting to changes on these
-resources. The cluster controller and the backup controller (Figure \ref{fig:design} C), though technically
+resources. MySQLBackupInstances (I\#1, I\#2) are used to keep track of already scheduled backups
+and should not be created manually, but the controller ensures that an actual backup file is
```

Comment:
Without "and should not be created manually" - this is mentioned above.
Without ", but", just start a new sentence, "The controller ensures..."